test/unit_tests/aievec_tests/bf16_exp_lut/testbench.cc

```diff
@@ -5,7 +5,14 @@
 #include <cstdio>
 #include <cstdlib>
 
+#ifdef TO_LLVM
```

Comment:
Why guard it? Does it cause issues with chess when it's declared as a C function?

Reply:
Short answer: yes. Currently the aievec->cpp flow sends `dut` through name mangling, while the aievec->llvm flow does not. Removing the guard causes the aievec->cpp tests to fail to link the mangled and unmangled `dut`.

Reply:
Are you saying that chess doesn't understand the `extern "C"` directive?

Reply:
Chess understands the `extern "C"` directive, but the definition of `dut` in the cpp produced by aievec->cpp is not marked `extern "C"`. Since the declaration in `testbench.cpp` differs from the definition in `dut.cc`, the linker fails with `Error in "testbench.cc", line 27: undefined symbol "dut"`.

Reply:
Got it! Thanks!